services/cadip/rs_server_cadip/api/cadip_search.py

```diff
@@ -107,3 +108,55 @@ def search_products(  # pylint: disable=too-many-locals
             status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
             detail=f"General failure: {exception}",
         ) from exception
+
+
+@router.get("/cadip/{station}/session")
+@apikey_validator(station="cadip", access_type="read")
+def search_session(
+    request: Request,  # pylint: disable=unused-argument
+    station,
+    id: Annotated[Union[str, None], Query(description="SessionID filter e.g 'S1A_20170501121534062343")] = None,
+    platform: Annotated[Union[str, None], Query(description="Satellite filter e.g 'S1A")] = None,
+    start_date: Annotated[Union[str, None], Query(description="Start time e.g. '2024-01-01T00:00:00Z'")] = None,
+    stop_date: Annotated[Union[str, None], Query(description="Stop time e.g. '2024-01-01T00:00:00Z'")] = None,
+):  # pylint: disable=too-many-arguments, too-many-locals
+    """Endpoint to retrieve list of sessions from any CADIP station.
+
+    Args:
+        station (str): CADIP station identifier (MTI, SGS, MPU, INU, etc)
+        id (str, list-like-str): Session identifier
+            (eg: "S1A_20170501121534062343" or "S1A_20170501121534062343, S1A_20240328185208053186")
+        platform (str, list-like-str): Satellite identifier
+            (eg: "S1A" or "S1A, S1B")
+        start_date (str): Start date of the time interval
+        stop_date (str): Stop date of the time interval
+
+    """
+    # Tbd - change list split with typing
```

Comment:
What does "Tbd" stand for, "to be done", "to be defined" ? Is there some work left to do here ?

Reply:
Comment can be ignored, splitting arguments based on type need a special function, a lot of work just to replace a "split".

Reply:
OK so can you remove the comment so we're not confused when reading it ?